data-protocols/ids/ids-core/src/main/java/org/eclipse/edc/protocol/ids/service/CatalogServiceImpl.java

```diff
@@ -39,19 +38,17 @@ public CatalogServiceImpl(
         this.contractOfferResolver = Objects.requireNonNull(contractOfferResolver);
     }
 
-    /**
```

Comment:
If you delete comments, please be sure to amend the one that is on the inteface method, otherwise we could lose information.

Reply:
It's documented on the interface, added a missing sentence there.